.github/workflows/js.yml

```diff
@@ -36,10 +36,10 @@ jobs:
           npx nx build @lightprotocol/zk-compression-cli --skip-nx-cache
 
       # Commented for breaking changes to Photon
-      - name: Run CLI tests
-        run: |
-          source ./scripts/devenv.sh
-          npx nx test @lightprotocol/zk-compression-cli
+      #- name: Run CLI tests
```

Comment:
Are they commented, because they are failing?

Reply:
yes, all tests which include photon are disabled because photon has the old addresses hardcoded.